helm/charts/hydra/templates/ingress-admin.yaml

```diff
@@ -1,7 +1,14 @@
 {{- if .Values.ingress.admin.enabled -}}
 {{- $fullName := include "hydra.fullname" . -}}
-{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+{{- if and .Values.ingress.admin.className (not (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion)) }}
```

Comment:
As discussed on slack, please simplify the setup and remove the logic for older versions of k8s, as 1.14 is already EOL ;)